### PR TITLE
New Jwebbinar notebook for cubeviz demo

### PR DESCRIPTION
**Pull request by PatrickOgle**
Notebook for cubeviz demo, based on a subset of the ifu_optimal notebook, with a better data set.

### CODE REVIEW

**Comment by camipacifici** [comment]
@PatrickOgle 
Trying the notebook now. A few comments:
- You will need more introduction to prepare the attendees for what they will see during your notebook. A table of content is a good way. Also, you can move the introduction before the imports.
- Data should be stored on box, not github. Ask me for the link if you cannot find it.
- There is a comment to install the pipeline. There should not be installation commands in these notebooks.
- Developer notes should not be included in these notebooks.
- I really like the Tasks well spelled out!
- Make sure you explain why you have try-except.
- Please provide the "live" version of the notebook that you will use to code during the webinar. Here is an example: https://github.com/spacetelescope/jwebbinar_prep/blob/main/pipeline_products_session/jwst-data-products-part1-live.ipynb

File: jdat_session/Cubeviz_Webbinar_solutions.ipynb [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Download the cube data from Box\n",
+    "filename = \"https://stsci.box.com/shared/static/ff0bj31acot1272x5qq2clbmto3s6v4f.fits\"\n",
+    "cube_file = download_file(filename)\n",
```
Comment (eteq) [inline jdat_session/Cubeviz_Webbinar_solutions.ipynb:117]:
```suggestion
    "cube_file = download_file(filename, cache=True)\n",
```

**Review by eteq — APPROVED** [review]
One minor suggestion!

File: jdat_session/Cubeviz_Webbinar_solutions.ipynb [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+      "  6  ASDF          1 BinTableHDU     11   1R x 1C   [32630B]   \n",
+      "\n",
+      "Wavelength:  [0.96652146 0.96675646 0.96699146 ... 1.89289145 1.89312645 1.89336145] um\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Download the cube data from Box\n",
+    "filename = \"https://stsci.box.com/shared/static/ff0bj31acot1272x5qq2clbmto3s6v4f.fits\"\n",
+    "cube_file = download_file(filename)\n",
```
Comment (eteq) [inline jdat_session/Cubeviz_Webbinar_solutions.ipynb:157]:
```suggestion
    "cube_file = download_file(filename, cache=True)\n",
```

**Pull request merged**